end-user-application-machines/azuredeploy.json

```diff
@@ -103,6 +103,12 @@
           "metadata": {
             "description": "PCoIP Standard Agent License Registration Code."
           }
+        },
+        "AgentChannel": {
+          "type": "string",
+          "metadata": {
+            "description": "from the channel to download PCoIP Agent Installer."
```

Comment:
Should use the description from the other template I think.

Reply:
@fwang-teradici  - sorry I'm being annoying but I think you missed this comment?